tests/test_pre_transform.py

```diff
@@ -137,9 +137,9 @@ def testSuccess(self):
         with self.assertLogs(logger=self.logger, level="INFO") as cm:
             update_quantum_graph(config, str(self.src), self.tmpdir)
         _, command = config.search("updateQuantumGraph", opt={"curvals": {"qgraphFile": str(self.src)}})
-        self.assertRegex(cm.output[0], r"[Bb]acking up")
-        self.assertRegex(cm.output[1], r"[Cc]ompleted")
-        self.assertRegex(cm.output[2], command)
+        self.assertRegex(cm.output[0], "[Bb]acking up")
```

Comment:
```suggestion
        self.assertIn("backing up", cm.output[0].lower())
```
Not required but something to consider if you don't want to use the regex for what is effectively a case insensitive search.